src/pybind/mgr/dashboard/frontend/src/app/shared/components/notification-banner/notification-banner.component.html

```diff
@@ -0,0 +1,9 @@
+<cd-alert-panel *ngIf="displayNotification"
+                class="no-margin-bottom"
+                [showTitle]="false"
+                size="slim"
+                [type]="notificationType"
```

Comment:
IMHO `notificationLevel` or `notificationSeverity` would be a more appropriate name.